awx/ui_next/src/screens/Job/Job.jsx

```diff
@@ -106,19 +105,23 @@ class Job extends Component {
         <Card>
           {cardHeader}
           <Switch>
-            <Redirect from="/jobs/:id" to="/jobs/:id/details" exact />
-            {job && (
+            <Redirect
+              from="/jobs/:type/:id"
+              to="/jobs/:type/:id/details"
```

Comment:
We'll need to update the launch button component to make sure it continues to route users to the correct location: https://github.com/ansible/awx/blob/devel/awx/ui_next/src/components/LaunchButton/LaunchButton.jsx#L45

Reply:
I added some Routes with a redirect component that uses the unified jobs api to do a lookup and redirect to the canonical url of a job. So `/jobs/123/details` will now redirect to e.g. `/jobs/playbook/123/details`